contributor-docs/development/tests/conformance-tests-sonobuoy.md

```diff
@@ -0,0 +1,220 @@
+---
+title: SCS Conformance Test KaaS Sonobuoy
+type:
+status: Draft
+track: Global
+---
```

Comment:
I know the other files in this directory have a front matter, but I don't think that we are actually using it in the docs pages. It can be dropped, and it should be dropped, because it could confuse people.